events/overview/track_attendees.rst

```diff
@@ -0,0 +1,46 @@
+====================
+Track your Attendees
+====================
+
+Attendees are at the heart of your event. Keeping track of their number and being able to manage
+their registration and attendance is necessary for planning and analyzes reasons.
```

Comment:
Attendees are at the heart of your event. Keeping track of their number and managing their registration and attendance is necessary for planning and analyzing reasons.